Main/src/main/java/com/main/acad/dao/Dao.java

```diff
@@ -1,18 +1,21 @@
-package main.java.com.main.acad.dao;
+package com.main.acad.dao;
+
+import com.main.acad.entity.Chapter;
 
-import main.java.com.main.acad.entity.Chapter;
 import java.util.List;
 
 public interface Dao {
 
-         void addChapter(Chapter chapter);
+    void addChapter(Chapter chapter) throws ClassNotFoundException;
+
+    void updateChapter(Chapter chapter) throws ClassNotFoundException;
```

Comment:
remove this exceptions from signature (catch them and rethrow runtime exceptions)